Package.swift

```diff
@@ -116,6 +125,22 @@ awsCCalPlatformExcludes.append("source/windows")
 awsCCalPlatformExcludes.append("source/darwin")
 #endif
 
+awsCChecksumsExcludes.append("source/intel/visualc")
+#if arch(arm64)
+// includes source/arm
+// TODO: look at the compiler flag in C
+awsCChecksumsExcludes.append("source/intel")
+awsCChecksumsExcludes.append("source/generic")
+#elseif arch(x86_64) || arch(i386)
+// include src/intel/asm
+awsCChecksumsExcludes.append("source/arm")
+awsCChecksumsExcludes.append("source/generic")
+#else
+// includes source/generic
+awsCChecksumsExcludes.append("source/arm")
+awsCChecksumsExcludes.append("source/intel")
+#endif
```

Comment:
We can’t exclude everything and specify what to include because [sources](https://developer.apple.com/documentation/packagedescription/target/sources) property recursively search for source files. It will include everything under `source/` and we will need to exclude as we are doing here.

Reply:
Even if we use sources and tell it to include some files, the Swift compiler still complains to explicitly exclude them.
` found 20 file(s) which are unhandled; explicitly declare them as resources or exclude from the target`